### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by nkbhasker**

### CODE REVIEW

**Review by dhawani — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -31,19 +34,86 @@ pnpm add @scalekit-sdk/node
...
+    {
+      state: "state",
+      connectionId: "connection_id",
+    }
+  );
+
+  res.redirect(authUrl);
+});
+
+// Handle the callback from the Scalekit
```
Comment (kunban-sk) [inline README.md:87]:
```suggestion
// Handle the callback from Scalekit 
```

File: README.md [hunk 2]
```diff
@@ -31,19 +34,86 @@ pnpm add @scalekit-sdk/node
...
+
+  res.redirect(authUrl);
+});
+
+// Handle the callback from the Scalekit 
+app.get("/auth/callback", async (req, res) => {
+  const code = req.query.code as string;
+  const authResp = await sc.authenticateWithCode(code, redirectUri);
+  res.cookie("access_token", authResp.accessToken);
+  res.json(token);
```
Comment (kunban-sk) [inline README.md:92]: this variable `token` here is undefined

File: lib/connect.js [hunk 3]
```diff
@@ -12,8 +12,8 @@ class GrpcConnect {
                 (next) => {
                     return (req) => {
                         req.header.set("User-Agent", this.coreClient.userAgent);
-                        req.header.set("x-sdk-version", this.coreClient.sdkVersion);
-                        req.header.set("x-api-version", this.coreClient.apiVersion);
+                        req.header.set("X-Sdk-Version", this.coreClient.sdkVersion);
+                        req.header.set("X-Api-Version", this.coreClient.apiVersion);
```
Comment (kunban-sk) [inline lib/connect.js:16]: Better to move the header keys in a constant

File: lib/organization.d.ts [hunk 4]
```diff
@@ -61,4 +60,17 @@ export default class OrganizationClient {
      * @returns {Promise<Link>} The admin portal link object with expiration time and location
      */
     generatePortalLink(organizationId: string): Promise<Link>;
+    /**
+     * Get admin portal link for an organization
```
Comment (kunban-sk) [inline lib/organization.d.ts:64]:
```suggestion
     * Get admin portal links for an organization
```

File: src/connection.ts [hunk 5]
```diff
@@ -65,5 +65,43 @@ export default class ConnectionClient {
...
+   * @param id The connection id
+   * @returns {Promise<ToggleConnectionResponse>} The connection enable response
+   */
+  async enableConnection(organizationId: string, id: string): Promise<ToggleConnectionResponse> {
+    return this.coreClient.connectExec(
+      this.client.enableConnection,
+      {
+        id,
+        identities: {
+          case: 'organizationId',
```
Comment (kunban-sk) [inline src/connection.ts:81]: lets create constant/enum for the case values

**Pull request merged**